ibis/backends/dask/execution/arrays.py

```diff
@@ -34,10 +35,23 @@
 )
 
 
-@execute_node.register(ops.ArrayColumn, tuple)
+@execute_node.register(ops.Array, tuple)
 def execute_array_column(op, cols, **kwargs):
-    cols = [execute(arg, **kwargs) for arg in cols]
-    df = dd.concat(cols, axis=1)
+    vals = [execute(arg, **kwargs) for arg in cols]
+    length = next((len(v) for v in vals if isinstance(v, dd.Series)), None)
+    n_partitions = next((v.npartitions for v in vals if isinstance(v, dd.Series)), None)
+
+    def ensure_series(v):
+        if isinstance(v, dd.Series):
+            return v
+        else:
+            return dd.from_pandas(pd.Series([v] * length), npartitions=n_partitions)
+
+    if length is None:
```

Comment:
nit: move this to directly beneath where we set `n_partitions =`, so that `ensure_series()` is only defined if it is going to get used.